### PR TITLE
Add new station trait: Pax Field

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Nanotrasen are testing a long range amplified pacification system in the sector, maybe it'll be tested on your station in a coming shift. /:cl:

With this, decidedly neutral, station trait, everyone on the station will be a pacifist for the first 5 to 15 minutes. How long will it take for people to notice? Almost immediately, let's be honest.

Due to limitations of the current pacifism system, this can be bypassed early if you do brain swap to a different mob shenanigans, but that's a lot of effort. Ideally, in the future, pacifism will be stored on the mind (or at least, be perfectly valid also on the mind), and then we can close that gap.

This is the sort of minor difference that will make the start of the shift's story fold out differently. It's a colourful difference, that people will howl about on Common, and lead to a series of non-lethal tabling scuffles. Eventually, the round violence will resume, but at least the janitors will have less to do.

---

- [ ] Should only affect mobs on the station Z level, otherwise I will be lynched by a bunch of angry miners who can't shoot fauna. (Unless that's EMERGENT GAMEPLAY?)
- [ ] Should be applied to abductors, wizards, nuke ops, (what about rat kings?), otherwise they'll have a field day with a station that can't fight back

### CODE REVIEW

**Comment by Kubisopplay** [comment]
Miners can't lynch you if they are paxed stupid!

**Comment by necromanceranne** [comment]
> Miners can't lynch you if they are paxed stupid!

They also can't mine except for picks and PKC's, so they've got a about 15 minutes to stew on things.

**Comment by Livrah** [comment]
Thank God you still can OOC - Suicide with Pax.

File: code/datums/station_traits/neutral_traits.dm [hunk 1]
```diff
@@ -320,3 +320,50 @@
...
 	greyscale_config = /datum/greyscale_config/festive_hat
 	greyscale_config_worn = /datum/greyscale_config/festive_hat_worn
+
+#define PAX_FIELD_LENGTH_MIN (5 MINUTES)
+#define PAX_FIELD_LENGTH_MAX (15 MINUTES)
+
+/datum/station_trait/pax
+	name = "Pax Field"
+	trait_type = STATION_TRAIT_NEUTRAL
+	weight = 5
```
Comment (Mothblocks) [inline code/datums/station_traits/neutral_traits.dm:330]: I don't see a world where this isn't 1 or maybe 2

**Comment by Zytolg** [comment]
There's no way a passive station can fail guys.

**Comment by MrDoomBringer** [comment]
This is fantastic

**Comment by ATH1909** [comment]
miners get fucked over, I guess

**Comment by MrDoomBringer** [comment]
> miners get fucked over, I guess

It says in the PR description that this will only affect mobs on the station z-level

**Comment by san7890** [comment]
i don't think this is a good idea.

First point: it always seems to be quite rewarding when you're able to pop down and kill the guy that you're angling for due to your objectives before they get the opportunity to get 3 fire extinguishers, five bars of soap, two flashes, etc... a time when they're vulnerable. this defacto "invulnerability" period doesn't really mesh well with that concept... it kinda makes the game feel a _lot_ less dangerous. 

Cults are most effective in those vulnerable minutes, and if they roll with this station trait, their entire shift is _completely_ bust since they will never get beyond those starting three (cult spells may not check for pacifism, but it ruins so much creativity when it comes to taking down potential convertees).

this is also very skewed against miners (unless they lose pacifism when they leave the z-level?), anyone on kilostation or tramstation (hostile mobs in maints), anyone who goes into space to set up solars and gets accosted by a carp, etc. Touching more on that, a carp can spawn during your period of pacifism but after you've applied the status effect to everyone. this would mean that you would get literally fucking assblasted by a carp. when you consider how many hazards people bake into just the station this seems more and more prone to just being completely unfair to people at the complete start of a round (let alone the first point I made)

there's just too many balance considerations for me to really agree with this. pacifism in its whole is like 90% implemented because people play it, run into a few bugs, and then realize that they can't engage with the entire game because we require an aggro grab here and there- and abandon it. a lot of people i've seen play pacifists over the years tend to focus more on the roleplay aspect of the game (not a bad thing!) but they do not engage with much of any in-depth mechanics. i would say that this would be an excellent way to weed out pacifism bugs, but it comes at a very steep cost.

also as a hypothetical: the max time is too long. if it gets all the way up to 15 minutes and everyone hates the shift because they couldn't do what they wanted to do, they will call at ~20 minutes. the round will be over before the half-an-hour mark. if i wasn't closing this, i would say make the max shorter

it also makes no sense thematically to be honest (which is important for station traits). you don't run an experiment for a few minutes at a time, you do it throughout a shift to collect maximal data. if they were firing it for a few minutes at a time every shift, it would shatter whatever "continuity" you would have from shift-to-shift-to-shift because they would obviously know that it worked- and i think a megastellar corporation would really like their employees to be docile always

sorry for the wall of text. it's a novel idea and i would really like to see it if i didn't think it completely clashed with the above... four-and-a-half points.

**Comment by MrDoomBringer** [comment]
TBH rather than a station trait, this might work better as a rare station-z-only random event. 

Roundstart pax means it's either unnoticed by most players or fucks over cults. Midround means it'll be more likely to spice up existing conflict among equal parties. Would make for much more interesting stories IMO. 
@san7890 thoughts? Aside from the lore fluff that can always be changed

**Comment by san7890** [comment]
i just fucking _hate_ pacifism and don't really like seeing it extended without a proper reason beyond "eh might be interesting"

one thought behind adding this as a trait would be that it could be used as an admin tool in order to facilitate nations setup or something like that, having a grace period where pacifism is turned off... i think that is the only instance I would be "okay" with having a pacifism field or something like that. as an admin tool that has to be consciously turned on/off for some ulterior reason. that's the only time/capacity i see it as being fitting to this game: because it's technically an event and would be a useful tool to have. 

hope that helps. i am and always will be only okay with pacifism being a conscious decision by someone. the game is just so gimped if the person doesn't knowingly enter into pacifism, or if it isn't forced onto them by someone who knowingly/deliberately doing it (admin doing an event or something). random pacifism (let alone mass pacifism) sucks balls

**Pull request closed without merging**